includes/Core/Admin_Bar/Admin_Bar.php

```diff
@@ -140,13 +140,8 @@ public function is_active() {
 			return false;
 		}
 
-		$current_screen = function_exists( 'get_current_screen' ) ? get_current_screen() : false;
-		if (
-			is_admin() &&
-			( false === $current_screen || 'post' !== $current_screen->base ) ||
-			( false !== $current_screen && 'post' === $current_screen->base && 'add' === $current_screen->action )
-		) {
-			return false;
+		if ( $this->is_admin_post_screen() ) {
+			return true;
```

Comment:
This is not right. The `$this->is_admin_post_screen()` condition should replace the `is_admin()` condition below.